docs/02-requirements/LZ-02-03.adoc

```diff
@@ -3,12 +3,15 @@
 [[LG-02-03]]
 ==== LZ 02-03 [ehemaliges LZ 4-1]: Qualitäten eines Softwaresystems verstehen und erklären (R1)
 
+Softwarearchitekt:innen wissen dass der Begriff "Qualität" in verschiedenen Kontexten unterschiedlich benutzt wird: 
+
+** im Kontext von Qualitätsmanagement im Sinne von "Güte" und 
+** im Sinne spezifischer Eigenschaften (eines Softwaresystems). 
```

Comment:
Is there a reason why you used double asterisks here? I think one per bullet is fine 🙂

Reply:
right u are

Reply:
done with cca185315ffd5fc70aa55d1ba5bfcc99db034a6f